dm/tests/dmctl_advance/check_list/binlog.sh

```diff
@@ -13,13 +13,20 @@ function binlog_invalid_binlogpos() {
 		"\[.*\], Message: invalid --binlog-pos $binlog_pos in handle-error operation: the pos should be digital" 1
 }
 
-function binlog_invalid_sqls() {
+function binlog_invalid_sqls_by_replace() {
 	sqls="alter table tb add column a int; alter table tb2 b int;"
 	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"binlog replace test-task $sqls" \
 		"invalid sql" 1
 }
 
+function binlog_invalid_sqls_by_inject() {
```

Comment:
do we also add a it test for `handle-error list`?

Reply:
ok, i will add more integration test.